src/build/plugins/api-docs/typedoc.js

```diff
@@ -88,6 +89,8 @@ export async function generateTypeDocJSON({ packageName }) {
   if (project) {
     let data = typedocApp.serializer.projectToObject(project, typeInfo.dir);
 
+    writeFileSync(packageName + '-project.json', JSON.stringify(data, null, 2));
```

Comment:
oops, forgot to remove this debug thing @NullVoxPopuli

Reply:
oh I thought you were actually using it for something :see_no_evil: